packages/projects-docs/pages/learn/editors/desktop-interface.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Desktop Interface
+description:
+---
+
+import { Callout } from 'nextra-theme-docs'
+import Video from '../../../../../shared-components/Video'
+
+# VS Code Desktop
+
+![](../images/vscode-desktop.png)
+
+Develop any repository or Devbox in your VS Code Desktop app and enjoy uninterrupted use of all the extensions and keybindings you’ve already configured.
+The CodeSandbox extension provides seamless interaction with all core features allowing you to retain the efficiency and collaboration of CodeSandbox in the comfort of your own setup.
+
+<br/>
+## Open your project from the web editor
+
+You can jump to VS Code at anytime from the browser, as shown below.
+
+![Open VS Code from Web Editor](../images/vscode-desktop-open.png)
+
+> We also support opening the branch using [Visual Studio Insiders](./insiders).
+
+## Open your project from VS Code Desktop
+
+Once the extension is installed, you can open a Devbox or repo branch from the Dashboard section, as shown below.
+
+![open from desktop](../images/vscode-desktop-dashboard.png)
+
+You will only be able to select from a list of repositories that have been imported to CodeSandbox. If you don’t see your project in the list, check your dashboard on CodeSandbox to make sure everything is set up there first.
+
+If, at any point, you want to switch to the CodeSandbox editor, simply click ‘Open in CodeSandbox’ under the ‘Control Panel’.
+
+<br/>
+## Control Panel
+
+The control panel contains actions for managing branches, opening a PR, and evaluating VM usage.
+
+![VS Code DevTool Panel](../images/vscode-desktop-control-panel.png)
+
+<br/>
+## DevTools
+
+Just like in the CodeSandbox editor, you can access running ports to view code changes in the browser. The available ports are listed in the panel. Clicking on a port will open a tab in your default browser.
+
+![VS Code DevTool Panel](../images/vscode-desktop-devtools.png)
+
+<br/>
+## Live Collaboration
+Collaboration is one of the core CodeSandbox features. Users who are active on a devbox or repository branch are on the same VM. That means that all the changes made by users active on the VM will be visible in real-time.
+
+<br/>
+### View collaborators
+
+Once you are connected to the branch, you will be able to see a list of teammates who are active on the branch. You can also see the list of collaborators and the environment they are working from.
+
+![Participant list](../images/vscode-desktop-participants.png)
```

Comment:
@necoline This image is missing.

Reply:
It was an issue with the image file name. I fixed it